ddtrace/constants.py

```diff
@@ -0,0 +1 @@
+PROCESSING_PIPELINE_KEY = 'PROCESSING_PIPELINE'
```

Comment:
@bmermet what about calling it just `FILTERS`? Considered that the main usage of the pipeline is to filter traces (or spans)